src/shared-components/accordion/index.tsx

```diff
@@ -16,6 +16,10 @@ export interface AccordionProps {
   disabled?: boolean;
   /** determine if the accordion is collapsed (false) or expanded (true) */
   isOpen: boolean;
+  /** show if the photo is required */
+  isRequired?: boolean;
+  /** when true, shows Required/Optional tag */
+  showRequiredTag?: boolean;
```

Comment:
I might change the name of `showRequiredTag` to something like `displayRequiredOrOptionalText`.

The naming as is feels too similar to `isRequired`. Even with the comment block, I didn't fully understand the point of having both variables until looking at the actual code that references them.

Reply:
good idea, I'll make the change!